lib/common/socket/evloop.c.h

```diff
@@ -19,11 +19,58 @@
  * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
  * IN THE SOFTWARE.
  */
+
+#include <stdlib.h>
+#ifdef _WIN32
+
+struct iovec {
+   char *buf;
+   unsigned long len;
+};
+
+static ssize_t
+writev(SOCKET fd, const struct iovec *iov, int iovcnt) {
```

Comment:
For simplicity please do the following:
* change the invocations of `writev` casting the second argument to `struct iovec *` to casting to `void *`
* change the type of the second argument in the definition of `writev` for WIN32 to `h2o_iovec_t *`
* remove the definition of `struct iovec`